doc/user-guides/acrn-sanity-test.rst

```diff
@@ -0,0 +1,342 @@
+#. Check ‘cat /proc/meminfo’ and see if the available memory is close the size passed through to UOS
+
+Expected Result
+
+1. Boot to UOS successfully
+#. Step 3: UOS - Depends on parameter passed to acrn-dm in launch_uos.sh.
+
+ST_HV_DM_LaaG_DefaultCPU_2GMemory_Conf
+**************************************
+
+* Description
+
+This test case focus on check if the pass through memory and CPU with certain 
+value in launch_uos.sh are correct.
+
+* Prerequisites
+
+Service OS and Guest OS boot up normally
+
+* Test Steps
+
+1. Boot to SOS
+#. Create UOS with 1core and memory=2048
+#. Check “cat cat /proc/cpuinfo” and “cat /proc/meminfo”
+
+* Expected Result
+
+1. Boot to UOS successfully
+#. Step 3: UOS memory size and CPU cores depends on parameter passed to acrn-dm in launch_uos.sh.
+
+IO
+**
+
+ST_ACRNIO_LaaG_Virtio_net
+*************************
+
+* Description
+
+This test case focus on checking if Service OS and LaaG can be allocated different IP, 
+and ping is ok by sharing same physical network card.
+
+* Prerequisites
+
+Need support Virtio, and bridge/tap is created normally in Service OS
+
+* Test Steps
+
+1. Boot DUT, and enter SOS from acrn hypervisor
+
+   .. code-block:: none
+
+      $ vm_console
+
+#. Check IP and bridge/tap after log in sos
+
+   .. code-block:: none
+     
+      $ ip a
+      $ brctl show
+      bridge name     bridge id               STP enabled     interfaces
+      acrn-br0         8000.1ad4ac453b20       no              acrn_tap0
+                                                               ens4
+
+#. Boot UOS using the launch_uos.sh 
+
+   .. code-block:: none
+   
+      $./launch_uos.sh
+
+#. Check UOS IP address
+
+* Expected Result
+
+IP address should be allocated for both SOS and UOS, and ping is ok
+
+ST_ACRNIO_LaaG_Virtio_storage
+*****************************
+
+* Description
+
+This test case focus on checking if LaaG using virtio storage can boot up successfully.
+
+* Prerequisites
+
+Need support Virtio
+
+* Test Steps
+
+1. In SOS: copy or create one img file as rootfs.
+#. Boot UOS using this img file in cmd line.
+
+   .. code-block:: none
+
+   -s 4,virtio-blk,/data/android/android.img \ (for android)
+   -s 3,virtio-blk,/data/clearlinux/clearlinux.img \  (for linux)
+
+* Expected Result
+
+UOS can boot up successfully
+
+ST_ACRNIO_SOS_Ethernet
+**********************
+
+* Description
+
+This test case focus on checking if IP can be allocated to Service OS correctly.
+
+* Prerequisites
+
+Network card is installed
+
+* Test Steps
+
+1. Boot into SOS
+#. IP a in SOS
+
+* Expected Result
+
+#. IP address (either local or external) should be allocated for sos(1 ip address), and ping is ok.
+
+ST_ACRNIO_SOS_Storage
+*********************
+
+* Description
+
+This test case focus on checking if EMMC can work normally in Service OS.
+
+* Prerequisites
+
+EMMC is installed
+
+* Test Steps
+
+1. Boot into SOS
+#. fdisk -l in SOS
+#. Find EMMC block and mount it to read/write /dev/mmcblk1p3
```

Comment:
please specify the hardware platform for emmc RW. Different hardware platforms have different mount dirs.